source/dom.js

```diff
@@ -1,4 +1,10 @@
-const div = document.createElement('div');
+import { isBrowserEnv } from './utils';
+
+const fakeWindow = {};
+
+export const global = isBrowserEnv ? window : fakeWindow;
+
+const div = isBrowserEnv ? document.createElement('div') : {};
```

Comment:
https://github.com/ilyabirman/Likely/blob/50b589cc9711545b0b5e62b97cb9318790cbe39a/source/button.js#L174-L176 — the only place of usage `createNode`.

Seems `createNode` could create a new Element every time which will be safely removed by GC, not big deal. I don't see any performance issue or something to create exactly one div inside the closure.

@iamakulov what do you think?